### PR TITLE
zh_trans:kubeadm_alpha_phase_controlplane.md

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.13 Features: set Milestone to 1.13 and Base Branch to dev-1.13
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_controlplane.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+### Synopsis
+
+
+This command is not meant to be run on its own. See list of available subcommands.
+
+### Options
+-->
+
+生成建立control plane所需的所有静态Pod清单文件
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_controlplane.md]: control plane 直接翻译成控制面好了

Comment (markthink) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_controlplane.md]: 我觉得控制面也是可以的，中英文处理建议使用软件处理一下，比如空格..
参考说明：https://github.com/k8smeetup/k8s-official-translation

**Pull request merged**